pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonDetails.java

```diff
@@ -114,15 +116,16 @@ public void appendExecutableDirectory(File file) {
         searchPath.add(file);
     }
 
-    public void setPythonVersion(String pythonVersion) {
-        if ("2".equals(pythonVersion)) {
-            pythonVersion = "2.6";
-        }
+    public void setPythonDefaults(PythonDefaults defaults) {
+        pythonDefaults = defaults;
+    }
 
-        if ("3".equals(pythonVersion)) {
-            pythonVersion = "3.5";
-        }
+    public PythonDefaults getPythonDefaults() {
+        return pythonDefaults;
+    }
 
+    public void setPythonVersion(String version) {
+        pythonVersion = new PythonVersion(pythonDefaults.normalize(version));
```

Comment:
Right, but I think that's something that a project owner should define. If the project doesn't specify which versions are acceptable, then we shouldn't pick some defaults. @zvezdan might have an opinion on this.